rpc_jobs/rpc_maas.yml

```diff
@@ -1,3 +1,29 @@
+- project:
+    name: "rpc-maas-master-bionic-premerge"
+    repo_name: "rpc-maas"
+    repo_url: "https://github.com/rcbops/rpc-maas"
+    series: "master"
+    branches:
+      - "master.*"
+    skip_pattern: |
+      \.md$
+      | \.rst$
+      | ^releasenotes/
+      | ^doc/
+      | ^gating/generate_release_notes/
+      | ^gating/periodic/
+      | ^gating/release/
+    image:
+      - "bionic"
+    scenario:
+      - rocky:
+          SLAVE_TYPE: "nodepool-maas-ubuntu-bionic-g1-8"
```

Comment:
Can these tests pass in Phobos? If so, this can just be ``nodepool-ubuntu-bionic-g1-8``.